Update references to py-rcsb_exdb_assets from development to master

brindakv opened the pull request.

aliciaaevans commented on README.md:
```diff
@@ -516,21 +516,21 @@ site_info_configuration:
-  VRPT_DICT_MAPPING_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionaries/vrpt_dictmap.json
+  PDBX_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/mmcif_pdbx_v5_next.dic
+  RCSB_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/dist/rcsb_mmcif_ext.dic
+  IHMDEV_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_file/reference/ihm-extension.dic
```
```suggestion
  IHMDEV_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/ihm-extension.dic
```
typo

aliciaaevans replied:
Also, updating `HISTORY.txt` and the version in `__init__.py`. (unless that's not needed since no python code changed?)

aliciaaevans reviewed (changes requested):
Just one typo. Also `rcsb/db/config/exdb-config-example.yml` needs the update notes at the top of the file.

brindakv commented:
Thanks @aliciaaevans. I updated the header comment and fixed the typo. I am not sure if I should update `History.txt` and `__init__.py` here considering it is such a minor update.

brindakv commented:
Based on our discussion today, this update involves the README and a config file only. So no version update is needed.

piehld commented:
@brindakv Correct, and also need to update the mock-data.

brindakv commented:
Thanks @piehld. Updated mock-data.

brindakv commented:
I'll merge when the checks are completed.

piehld commented:
Sounds good @brindakv, thanks for the updates!
